src/libraries/System.Private.CoreLib/src/System/SearchValues/SearchValues.cs

```diff
@@ -18,6 +18,9 @@ namespace System.Buffers
     /// </remarks>
     public static class SearchValues
     {
+        internal static readonly SearchValues<char> WhiteSpaces = Create(
```

Comment:
Just remembered that we already have this defined in
https://github.com/dotnet/runtime/blob/5667c2887f607939e5ce054137481f877530a279/src/libraries/System.Private.CoreLib/src/System/String.Manipulation.cs#L38-L39